### PR TITLE
[feat] Tabs admin - basic set up

### PR DESCRIPTION
**Pull request by marcoviappiani**
This is allows to view the current tabs on the page /tabs
It's not a current reflection of where I'm at. Sending it because since there will eventually be a lot of files and changes, it should be split into multiple commits and pull requests.

This one represents the ability to view the existing tabs without having any actions enabled on them.

### CODE REVIEW

File: modules/core/client/controllers/adminTabs.controller.client.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+angular.module('tropicalbs')
+  .controller('AdminTabsController', AdminTabsController);
+
+  AdminTabsController.$inject =  ['tabsService'];
```
Comment (codydaig) [inline modules/core/client/controllers/adminTabs.controller.client.js]: Indentation

File: modules/core/client/routes/tabsAdmin.routes.client.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+angular.module('tropicalbs')
+  .config(tabsAdminRoutes);
+
+  tabsAdminRoutes.$inject = ['$stateProvider'];
```
Comment (codydaig) [inline modules/core/client/routes/tabsAdmin.routes.client.js]: Indentation

File: modules/core/client/views/adminTabs.view.client.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+           {{ tab.title }}
+         </div>
+         <div flex="10">
+           {{ tab.uisref }}
+         </div>
+         <div flex="60" layout="row" layout-align="start center">
+           <md-chips flex="100" readonly="true" ng-model="tab.visibleRoles">
+           </md-chips>
+         </div>
+<!--          <div flex="20">
```
Comment (codydaig) [inline modules/core/client/views/adminTabs.view.client.html]: Indentation

**Comment by codydaig** [comment]
LGTM

**Comment by RRDuke** [comment]
LGTM

**Pull request merged**